Helloworld.java

```diff
@@ -21,7 +37,133 @@ public static void main(String[] args) {
             else{
                     System.out.println("This progam will allow you to list content of directory that you can choose");
             }
-            switch(menu){
+        return menu;
+    }
+    public static String operatingSystemMenuBlock(boolean exitFromProgramCase,int os, String path){
+        for(;exitFromProgramCase==false;){
+            switch (os){
```

Comment:
Why the hell I have to choose OS manually? We definitely need to do this automatically